docker/config/default.config.envsubst

```diff
@@ -24,3 +24,6 @@ api.prometheus.bind_address=$RADIXDLT_PROMETHEUS_API_BIND_ADDRESS
 
 testing_forks.enable=$RADIXDLT_TESTING_FORKS_ENABLE
 testing_forks.fork_config_name=$RADIXDLT_TESTING_FORK_CONFIG_NAME
+
+consensus.use_genesis_for_validator_address=$RADIXDLT_CONSENSUS_USE_GENESIS_FOR_VALIDATOR_ADDRESS
```

Comment:
Does this need to go into docker file and onto docker compose?

Reply:
Ideally both of these, yes.
* For normal test nets, they can both be blank (defaults to using genesis for validator address, and self-defining as a validator if its key matches; else being a full node)
* For configuring new validators, you can just specify $RADIXDLT_CONSENSUS_VALIDATOR_ADDRESS (and it will default to using that)